src/renderer/components/pages/HomePage.tsx

```diff
@@ -191,6 +233,20 @@ export class HomePage extends React.Component<IHomePageProps, IHomePageState> {
     onSelectPlaylist(hof, route);
   }
 
+  private onFavoriteClick = () => {
+    const { central, libraryData, onSelectPlaylist } = this.props;
+    let hof = findFavoritePlaylist(central.playlists.playlists);
```

Comment:
"hof" stands for "Hall of Fame". Consider renaming it to something more fitting like "fav" or "favorites".